### ISSUE TITLE
ActiveField array to string error when CSS classes set as an array

### ISSUE THREAD

**Issue by kartik-v** [yiisoft/yii2#14773]
### What steps will reproduce the problem?
When a CSS class is passed as an array to `yii\widgets\ActiveField`, it throws an array to string error. Code sample below:

```php
use yii\helpers\Html;
use yii\widgets\ActiveForm;
use app\models\Demo;
$model = new Demo();

$form = ActiveForm::begin();
$options = [];
Html::addCssClass($options, ['css-1','css-2']);
echo $form->field($model, 'email', ['options'=>$options])->textInput();
ActiveForm::end();
```

### What is the expected result?
CSS classes should be allowed to be passed as an array to `ActiveField` or to `Html:addCssClass`

### What do you get instead?
Array to string conversion error.

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.13
| PHP version      | Any
| Operating system | Any

**Comment by klimov-paul** [comment]
This is incorrect usage as you attempt to pass HTML options to the field options, which are different things.
Input HTML options in your example should be passed to `textInput()` method:

```php
use yii\helpers\Html;
use yii\widgets\ActiveForm;
use app\models\Demo;
$model = new Demo();

$form = ActiveForm::begin();
$options = [];
Html::addCssClass($options, ['css-1','css-2']);
echo $form->field($model, 'email')->textInput($options); // see the difference!
ActiveForm::end();
```

**Comment by kartik-v** [comment]
I am not sure you understood the issue and cross check again.

The problem is initiated in the `ActiveForm::field` [method here](https://github.com/yiisoft/yii2/blob/master/framework/widgets/ActiveForm.php#L299) which supports passing all `ActiveField` component properties as the third parameter to `ActiveForm::field` method.

One should be able to pass `ActiveField::options` using the above method isn't it?

I know that one can pass options also via `textInput('options')` or other methods... but when setting by the above process you face an error right?

**Comment by klimov-paul** [comment]
Resolved by commit e69adc19a7473f219a9713404d8bbe017ed0c3ec
